crates/kgraph_utils/src/transformers.rs

```diff
@@ -86,6 +86,11 @@ impl IntoDirValue for ast::ConnectorChoice {
         Ok(dir::DirValue::Connector(Box::new(self)))
     }
 }
+impl IntoDirValue for api_enums::Country {
+    fn into_dir_value(self) -> Result<dir::DirValue, KgraphError> {
+        Ok(dir::DirValue::BillingCountry(self))
+    }
+}
```

Comment:
Remove this impl. `enums::Country -> DirValue::BillingCountry` is not a one to one mapping. It can be `DirValue::BusinessCountry` as well.